test/e2e/instrumentation/monitoring/cadvisor.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2015 The Kubernetes Authors.
+Copyright 2017 The Kubernetes Authors.
```

Comment:
should this test be under instrumentation or under node team?

cc @dchen1107  @tallclair @dashpole